x-pack/plugins/fleet/server/services/setup.ts

```diff
@@ -43,24 +47,34 @@ export interface SetupStatus {
 
 export async function setupFleet(
   soClient: SavedObjectsClientContract,
-  esClient: ElasticsearchClient
+  esClient: ElasticsearchClient,
+  // Allows for passing in config options rather than pulling from the `appContextService` - useful in tests
```

Comment:
@joshdover I added this notion of a config override parameter to our setup function, but we're not actually using it anywhere. Should this remain in this PR or should we address this elsewhere?

Reply:
If we're not going to write tests right now for this, we can add it later.